iphone/Classes/TiUITableView.m

```diff
@@ -2440,7 +2501,7 @@ - (void)keyboardDidChangeFrame:(NSNotification *)notification
 - (void)tableView:(UITableView *)ourTableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
   BOOL search = NO;
-  if (!allowsSelectionSet || ![ourTableView allowsSelection]) {
+  if ((!allowsSelectionSet || ![ourTableView allowsSelection]) && !editing) {
```

Comment:
Are you sure this doesn't break apps that have `editing: true`?

Reply:
Yeah. I verified it. It's working fine for  me. If you have any test case please let me know. Thanks!